lib/product-import.coffee

```diff
@@ -164,16 +164,14 @@ class ProductImport
         @_getWhereQueryLimit()
       )
       Promise.map(skuChunks, (skus) =>
-        new Promise (resolve, reject) =>
-          predicate = @_createProductFetchBySkuQueryPredicate(skus)
-          @client.productProjections
-          .where(predicate)
-          .staged(true)
-          .all()
-          .fetch()
-          .then ({ body: { results } }) ->
-            resolve(results)
-          .catch (err) -> reject(err)
+        predicate = @_createProductFetchBySkuQueryPredicate(skus)
+        @client.productProjections
+        .where(predicate)
+        .staged(true)
+        .all()
```

Comment:
Maybe we should use `perPage(100)` here instead of default 50. I guess we need to merge following PR first: https://github.com/sphereio/sphere-node-sdk/pull/230